crates/jstz_cli/src/account.rs

```diff
@@ -216,14 +216,18 @@ fn list_accounts(long: bool) -> Result<()> {
     Ok(())
 }
 
-async fn get_code(account: Option<AddressOrAlias>) -> Result<()> {
+async fn get_code(
+    account: Option<AddressOrAlias>,
+    network: Option<NetworkName>,
+) -> Result<()> {
     let cfg = Config::load()?;
 
+    println!("Getting code.. {:?}.", network);
```

Comment:
Some of our commands are designed to be pipe-able (in the case of `jstz account code ... > index.js`)

```suggestion
    debug!("Getting code.. {:?}.", network);
```